projects/netgrif-components-core/src/lib/navigation/navigation-drawer/abstract-navigation-drawer.component.ts

```diff
@@ -109,10 +109,10 @@ export abstract class AbstractNavigationDrawerComponent implements OnInit, After
     }
 
     toggle(): Promise<MatDrawerToggleResult> {
-        if (this._fixed && this._sideNav.opened) {
-            return Promise.resolve('open');
+        if (!this._sideNav.opened) {
```

Comment:
toggle should close the sideNav if it is currently opened and it should open the sideNav if it is currently closed.

This implementation only opens the sideNav if it is closed.

Reply:
@minop corrected